frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -25,9 +26,7 @@ export const ReportStreamHeader = () => {
 
     if (authState !== null && authState.isAuthenticated) {
         if (reportReceiver(authState)) {
```

Comment:
This can be `permissionCheck(PERMISSIONS.RECEIVER, authstate)) || permissionCheck(PERMISSIONS.PRIME_ADMIN, authstate))` since it'll automatically forward the first condition to `reportReceiver` and we want to ensure admins independent of any state receiver org can still access the nav item.

Reply:
It seems like this complexity should be contained in a single util function call `IsReceiver()`?

Reply:
@kevinhaube this is resolved now